input/data/2008/2008ApJ...679.1427A/tev-000014-sed.ecsv

```diff
@@ -0,0 +1,24 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
```

Comment:
Yes, it is this paper. I have a few more on which I am working on. I can do this, and add a comment. But I still think this is not the correct solution - I don't think it is a bad way to let the tools calculate this later.

(actually where e_ref should be is not straightforward, and mostly not mentioned in publications).